api/util/validationSchemas.js

```diff
@@ -17,3 +17,22 @@ exports.incidentQuery = yup.object({
 
   endDate: yup.string().trim(),
 });
+
+exports.incidentObject = yup.object({
+  tweet_id: yup.string().required().trim().min(5).max(10),
+  user_name: yup.string().required().trim().min(2).max(100),
+  desc: yup.string().required().trim().min(2).max(1000),
+  city: yup.string().trim().min(2).max(50),
+  state: yup.string().trim().min(2).max(50),
+  lat: yup.string().trim().min(2).max(50),
+  long: yup.string().trim().min(2).max(50),
+  title: yup.string().trim().min(2).max(50),
+  force_rank: yup.string().required().trim().min(2).max(50),
+  status: yup
+    .string()
+    .required()
+    .trim()
+    .matches(/(pending|approved|rejected)/i, { excludeEmptyString: true }),
+  confidence: yup.number(),
+  tags: yup.string().required().trim(),
```

Comment:
Missing date_created

Reply:
Also, isn't tweet_id called incident_id?

Reply:
tweet_id is the id of the tweet, incident_id is the primary key

Reply:
Fixed in latest commit 👍